sources/Directory.Packages.props

```diff
@@ -4,6 +4,7 @@
   </PropertyGroup>
   <!-- Runtime dependencies -->
   <ItemGroup>
+    <PackageVersion Include="AssimpNet" Version="4.1.0" />

```

Comment:
Is it really a runtime dependency? Does the engine depend on it and not just the build/editor?

Reply:
> Is it really a runtime dependency? Does the engine depend on it and not just the build/editor?

Just build/Editor. It is part of SILK.NET, C#/.net bindings to Assimp library. It replaces Autodesk fbx sdk. Its editor only to convert fbx and other 3D file formats to Stride assets

Reply:
In that case it will better to set it in a section below. So that we keep a list of runtime dependencies separate from the rest.

Reply:
+1 This, binding version is from 2018 ... (almost 6 years ago)

Reply:
On top of that, as `AssimpNet` doesn't target explicitly any modern .NET (Core) versions, we don't know if that would ever work on other platform than Windows.

Reply:
Done. 
Yes we used SILK.NET.Assimp. It was already referenced in .props. Removed the Assimp reference!

Reply:
Excellent!